### PR TITLE
Add link to Ollie's caledar to FAQ

### LINKED ISSUE #382: Update FAQ
Add visit link to FAQ

### PR DESCRIPTION
**Pull request by nikeshashar**
Closes #382

### CODE REVIEW

File: data/faqs.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@
       answer: "Paul Graham, the tech startup visionary behind <a href='https://www.ycombinator.com/' target='_blank'>YCombinator</a> coined the term Maker for those individuals that truly produce. The Makers Academy mission is to give every student the skills to call themselves a Maker."
 
     - question: Where does Makers Academy take place?
-      answer: "Makers Academy officially runs 9am to 6pm, Monday to Friday at the Makers Academy studio in Shoreditch at 50-52 Commercial St – right at the centre of the tech startup community, a few minutes away from Shoreditch High Street station and Google Campus. We welcome anyone who wants to come visit and see how we operate."
+      answer: "Makers Academy officially runs 9am to 6pm, Monday to Friday at the Makers Academy studio in Shoreditch at 50-52 Commercial St – right at the centre of the tech startup community, a few minutes away from Shoreditch High Street station and Google Campus. We welcome anyone who wants to <a href='http://www.calendly.com/ollieporter/15min' target='_blank'>come visit</a> and see how we operate."
```
Comment (spike01) [inline data/faqs.yml]: Would it make more sense for this to say "arrange a visit"?

**Comment by spike01** [comment]
:shipit: but read my comment

**Comment by pitchinvasion** [comment]
:shipit:

**Comment by nikeshashar** [comment]
@spike01 's comments made sense. Updated the text. @pitchinvasion ready to :ship: ?

**Pull request merged**